testing.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * A simple React component
+ */
+// What is happening here?
+class Why extends React.Component {
```

Comment:
Is this file part of the CFH codebase? It is on react and the codebase uses angular...